pkg/combustion/templates/script-base.sh.tpl

```diff
@@ -1,7 +1,17 @@
 #!/bin/bash
-# combustion: network
 set -euo pipefail
 
+{{ if .NetworkScript -}}
+# combustion: prepare network
+
+if [ "${1-}" = "--prepare" ]; then
+    ./{{ .NetworkScript }}
+    exit 0
+fi
+{{- else -}}
+# combustion: network
```

Comment:
This is so funny to me since it looks like an else block that simply lays down a comment, but the implications are enormous.